### ISSUE TITLE
Symbology color contrast for accessibility

### ISSUE THREAD

**Issue by ekamptner** [CityOfNewYork/poletop-finder#22]
I'm concerned that the current light blue used for symbology does not have high enough contrast to meet accessibility requirements.
Check out this tool -- https://webaim.org/resources/contrastchecker/

Our basemap grey is about #ececec

If we have time, can we see what  #2e6ba4 looks like for icons? If not, need to loop back with MOPD. 

Notes:
DoITT Theme
#2e6ba4 (blue), #3094d8 (light blue), and #404040 (dark gray)

**Comment by timkeane** [comment]
fixed in c26ab48 and 7dc3e65
